### PR TITLE
Completed feature/reviews-sections, also made responsive

### PR DESCRIPTION
**Pull request by SiddheshShinde-tech**

### CODE REVIEW

File: css/sections.css [hunk 1]
```diff
@@ -113,3 +168,74 @@
 }
 
 /* FREE DEMO BUTTON ENDS*/
+
+
+/* ******************** */
```
Comment (workeazee) [inline css/sections.css]: Add this in responsive.css

File: index.html [hunk 2]
```diff
@@ -172,141 +172,377 @@ <h1 class="main-heading">Work Flow</h1>
...
-            lobortis ligula rhoncus diam consectetur, vitae efficitur nibh
-            fringilla. Sed gravida porttitor molestie. Sed pellentesque lectus
-            sed lacinia accumsan. Sed condimentum arcu risus, non convallis eros
-            tincidunt et. Curabitur elementum mi vitae elit accumsan aliquet.
-            Nam pellentesque molestie pulvinar. Nullam id mauris posuere,
-            sagittis lectus quis, commodo ante. Lorem ipsum dolor sit amet,
-            consectetur adipiscing elit. Morbi faucibus eros metus, sit amet
-            volutpat lorem bibendum ac.
-          </p>
+        <h2 class="main-heading">Reviews</h2>
```
Comment (workeazee) [inline index.html]: Dont remove the `<div class="text-part">` from here. Keep the same format everywhere

File: index.html [hunk 3]
```diff
@@ -373,9 +609,19 @@ <h1 class="main-heading">Contact Us</h1>
 
         <div class="handlefooter">
           <div class="footer-icons">
-            <a target="_blank" href="mailto:workeazee@gmail.com"><i class="fas fa-envelope"></i></a>
-            <a target="_blank" href="https://api.whatsapp.com/send?phone=919769417465"><i class="fab fa-whatsapp"></i></a>
-            <a target="_blank" href="https://www.linkedin.com/showcase/janussoftwares"><i class="fab fa-linkedin-in"></i></a>
+            <a target="_blank" href="mailto:workeazee@gmail.com"
```
Comment (workeazee) [inline index.html:614]: Do shift+alt+f on all pages you commit

**Comment by SiddheshShinde-tech** [comment]
Amended the new changes.

File: index.html [hunk 4]
```diff
@@ -373,9 +611,19 @@ <h1 class="main-heading">Contact Us</h1>
 
         <div class="handlefooter">
           <div class="footer-icons">
```
Comment (workeazee) [inline index.html:613]: Why is the code changed here?

**Pull request merged**